packages/create-pantheon-decoupled-kit/src/templates/partials/shared/readme/sharedReadme.ts

```diff
@@ -0,0 +1,173 @@
+import { taggedTemplateHelpers as utils } from '@cli/utils';
+
+type GeneratorName = 'next-wp' | 'next-drupal' | 'gatsby-wp';
+type CmsType = 'drupal' | 'wordpress';
+
+/**
+ * The header section for the starter kit READMEs
+ */
+export const readmeHeader =
+	() => /* md */ `<div style="display:flex;flex-direction:column">
+	<img src="https://raw.githubusercontent.com/pantheon-systems/decoupled-kit-js/canary/web/static/img/B_Fist-Tagline.png" height="120" style="background:#ffffff;border-radius:8px;margin:auto;display:flex;" alt="Pantheon.io logo featuring a fist capturing lighting. Pantheon™, The Platform for Extraordinary Websites.">
+	<a href="https://pantheon.io">
+		<h1 style="margin:auto;" align="center">Pantheon</h1>
+	</a>
+</div>
+
+For detailed documentation on the Decoupled Kit Starters, visit [the Decoupled Kit developer documentation](https://decoupledkit.pantheon.io).
+
+For more information on using the starter on Pantheon Front-End Sites, visit [the Pantheon.io platform documentation](https://docs.pantheon.io/guides/decoupled/).`;
+
+/**
+ * The Getting Started section for the starter kit READMEs
+ * @param starterName one of Next WordPress, Next Drupal, or Gatsby WordPress
+ */
+export const gettingStarted = (
+	starterName: 'Next WordPress' | 'Next Drupal' | 'Gatsby WordPress',
+) => /* md */ `## Getting Started
+
+The ${starterName} starter requires Node.js and is built and tested on the LTS version which can be found on the [Node.js downloads page](https://nodejs.org/en/download).
+
+The starter kit supports npm, pnpm, and yarn. If you created the starer via the Pantheon Front-End Sites dashboard, it will use npm by default. To change this, delete the \`package-lock.json\` \`node-modules\` folder, then run the install command of your preferred package manager.
```

Comment:
This reads a little funny as rendered: delete the `package-lock.json node-modules` folder. Maybe something like:

delete the `package-lock.json` file and the `node-modules` folder